dev.php

```diff
@@ -19,14 +19,12 @@
  * GNU General Public License for more details.
  */
 
-add_action( 'init', function() {
+add_action( 'init', 'shortcode_ui_example' );
+
+function shortcode_ui_example() {
```

Comment:
Can we make this method `shortcode_ui_dev_example` to make it separate from the main plugin?